RecoParticleFlow/PFClusterProducer/src/LocalMaximumSeedFinder.cc

```diff
@@ -106,4 +114,8 @@ findSeeds( const edm::Handle<reco::PFRecHitCollection>& input,
       }
     }
   }
+
+  // auto ns = std::count(seedable.begin(),seedable.end(),true);
+  // std::cout << "seeds " << ns << std::endl;
```

Comment:
same as above: if this is needed, change to LogDebug or similar that wouldn't need code editing to enable